routes/books.js

```diff
@@ -10,9 +10,17 @@ router.get('/create', (req, res, next) => {
 router.post('/new', (req, res, next) => {
   const {title, author, image_url, description} = req.body
   db.createBook(title, author, image_url, description)
-    .then(book => res.redirect('/') )
-    .catch(error => { res.render('error', {error})
+    .then(book => res.redirect(`/books/${book.id}`) )
+    .catch(error => { res.render('error', {error})})
+})
+
+router.get('/:id', (req, res, next) => {
+  const book_id = req.params.id
+  db.getBookById(book_id)
+  .then(book => {
```

Comment:
Indentation off, and I'd move this up to one line since there's so little logic here.